internal/pkg/manifest/validate_env.go

```diff
@@ -81,6 +84,56 @@ func (cfg environmentVPCConfig) validate() error {
 	return nil
 }
 
+// validate returns nil if securityGroupRule has all the required parameters set.
+func (cfg SecurityGroupRule) validate() error {
+	if cfg.CidrIP == "" {
+		return fmt.Errorf(`cidr`)
```

Comment:
? can we instead return like this:
```suggestion
		return &errFieldMustBeSpecified{
				missingField: "cidr",
			}
```
It's very strange to read an error message that just returns the `"cidr"`